src/ranged.cpp

```diff
@@ -341,29 +424,52 @@ dealt_projectile_attack Creature::projectile_attack( const projectile &proj_arg,
     return attack;
 }
 
+static double erfinv( double x )
```

Comment:
Once your unit test is passing you can then write a non-iterative version? Disclaimer being that I haven't investigated the maths for that yet